README.md

```diff
@@ -68,6 +68,9 @@ Creates a new instance of `CliPie`.
  - `flat` (Boolean): If `true`, flat colors will be used (default: `true`).
  - `chr` (String): The character to draw the pie (default: `" "`).
  - `no_ansi` (Boolean): If `true`, ansi styles will not be used.
+ - `legend` (Boolean): If `true`, a legend is added next to the pie.
```

Comment:
Can you add these in the JS file as well? The ones from the readme file are generated actually. In the JS side we have JSDoc comments.